lib/modules/platform/github/graphql.ts

```diff
@@ -41,7 +41,7 @@ export const getIssuesQuery = `
 query(
   $owner: String!,
   $name: String!,
-  $user: String!,
+  $user: String,
```

Comment:
Does it work to just leave it undefined?

Reply:
Yes